apps/connect/src/context/reactor-analytics.tsx

```diff
@@ -88,6 +119,7 @@ export function ReactorAnalyticsProvider({ children }: PropsWithChildren) {
             }}
         >
             <DiffAnalyzerProcessor />
+            <DriveAnalyticsProcessor />
```

Comment:
Let's use a feature flag for this. Like `PH_CONNECT_DRIVE_ANALYTICS_ENABLED`, should be `false` by default.

Reply:
fixed in [f5ea412](https://github.com/powerhouse-inc/powerhouse/pull/1607/commits/f5ea412449650045077618d754b86e2c9756a952)